website/docs/r/log_analytics_workspace.html.markdown

```diff
@@ -45,6 +45,10 @@ The following arguments are supported:
 
 * `daily_quota_gb` - (Optional) The workspace daily quota for ingestion in GB.  Defaults to -1 (unlimited).
 
+* `enable_ingestion_over_public_dns` - (Optional) Should the Log Analytics Workflow support ingestion over the Public Internet? Defaults to `true`.
```

Comment:
given the description here should this be
```suggestion
* `internet_ingestion_enabled?` - (Optional) Should the Log Analytics Workflow support ingestion over the Public Internet? Defaults to `true`.
```

(we always put enabled at the end not the start)